ddl/table.go

```diff
@@ -481,6 +481,46 @@ func (w *worker) onRecoverTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver in
 	return ver, nil
 }
 
+func (w *worker) recoverTable(t *meta.Meta, job *model.Job, recoverInfo *RecoverInfo) (ver int64, err error) {
```

Comment:
The code is quite similar to `onRecoverTable`, could we make the common code shared?

Reply:
Done. PTAL ,Thx~